ui/apps/platform/src/hooks/useURLParameter.test.tsx

```diff
@@ -31,6 +31,17 @@ const createWrapper = (props) => {
     };
 };
 
+beforeAll(() => {
+    jest.useFakeTimers();
+});
+
+function actAndRunTicks(callback) {
+    return act(() => {
+        callback();
+        jest.runAllTicks();
+    });
+}
+
```

Comment:
Since the updates here now depend on the completion of a microtask, we need to use jest fake timers and exhaust the current task at the end of each `act` call to get the most up to date state value.